frontend/src/test/E_MIB.test.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import { shallow } from "enzyme";
+import E_MIB from "../components/eMIB/E_MIB";
+
+it("renders without crashing", () => {
+  const div = document.createElement("div");
+  ReactDOM.render(<E_MIB />, div);
+  ReactDOM.unmountComponentAtNode(div);
+});
+
+it("renders initial message", () => {
+  const wrapper = shallow(<E_MIB />);
+  const initialMessage = <p>Welcome to the eMIB Sample Test</p>;
+  expect(wrapper.contains(initialMessage)).toEqual(true);
+});
```

Comment:
It's be great to have tests when you force a state change in your test, and then check that the proper sub-component displays. For example, if state is `howTo`, does the `HowTo` component display?

Here's an example: https://stackoverflow.com/questions/45598423/react-enzyme-how-to-test-for-subcomponent

And here's an enzyme cheatsheet that I find useful: https://devhints.io/enzyme